src/audio/sound_file.cpp

```diff
@@ -243,6 +286,20 @@ std::unordered_map<std::string, std::string> fallback_paths = {
   {"leveldone.ogg", "/music/misc/leveldone.ogg"},
   {"theme.music", "/music/misc/theme.music"},
   {"theme.ogg", "/music/misc/theme.ogg"},
+  {"cave_old.music", "/music/retro/cave_old.music"},
+  {"cave_old.ogg", "/music/retro/cave_old.ogg"},
+  {"classic.music", "/music/retro/classic.music"},
+  {"classic.ogg", "/music/retro/classic.ogg"},
+  {"fortress_old.music", "/music/retro/fortress_old.music"},
+  {"fortress_old.ogg", "/music/retro/fortress_old.ogg"},
+  {"ice_music.music", "/music/retro/ice_music.music"},
+  {"ice_music.ogg", "/music/retro/ice_music.ogg"},
+  {"worldmap_old.music", "/music/retro/worldmap_old.music"},
+  {"worldmap_old.ogg", "/music/retro/worldmap_old.ogg"},
+  {"saharan_penguin.music", "/music/tropical/saharan_penguin.music"},
+  {"saharan_penguin.ogg", "/music/tropical/saharan_penguin.ogg"},
+  {"tropicalbreeze.music", "/music/tropical/tropicalbreeze.music"},
+  {"tropicalbreeze.ogg", "/music/tropical/tropicalbreeze.ogg"},
```

Comment:
Do we actually need to update this list, as it is used only for legacy music to work correctly?

Reply:
Can we fix the behaviour instead, so the `fallback_paths` are used if and only if the actual file isn't found?